x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -62,9 +68,39 @@ describe('Alerts timeline', () => {
   });
 
   it('Add an empty property to default timeline', () => {
+    // add condition to make sure the field is empty
+    openAddFilterPopover();
+    fillAddFilterForm({ key: 'file.name', operator: 'does not exist' });
     scrollAlertTableColumnIntoView(ALERT_TABLE_FILE_NAME_HEADER);
     addAlertPropertyToTimeline(ALERT_TABLE_FILE_NAME_VALUES, 0);
     openActiveTimeline();
     cy.get(PROVIDER_BADGE).first().should('have.text', 'NOT file.name exists');
   });
 });
+
+describe('Bulk Investigate in Timeline', () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule());
+    esArchiverLoad('bulk_process');
+  });
+
+  after(() => {
+    esArchiverUnload('bulk_process');
+  });
+
+  beforeEach(() => {
+    visit(ALERTS_URL);
+    waitForAlertsToPopulate();
+  });
+
+  it('Adding multiple alerts to the timeline should be successful', () => {
+    // select all visible events
+    investigateFirstPageEventsInTimeline();
```

Comment:
Assertions on tests should not be wrapped inside the tasks methods, should be explicitly written in the test.

Reply:
@MadameSheema .. Sure, I have now removed the assertions from tasks. Please check new updates.